packages/core/event.ts

```diff
@@ -53,6 +59,11 @@ export function getEventName(eventNameObj: EventNameObjectType, forAttendeeView
     eventName = eventName.replace("{LOCATION}", locationString);
   }
 
+  const duration =
+    startTime && endTime
+      ? `${dayjs(endTime).diff(dayjs(startTime), "minute")} mins`
+      : `${String(eventNameObj.eventDuration)} mins`;
+
```

Comment:
We are not fixing the actual bug; instead, we are adding a solution on top of it just to show the event duration. I don't think this is a good way to fix it. It used to work before. Could you check that more deeply?